styleguide/derek/global-components/filter-bar/filter-bar.js

```diff
@@ -1,13 +1,63 @@
+/* eslint no-use-before-define: 0 */
+
+const Zoolander = Zoolander || {};
+
+Zoolander.rsFilterBar = {
+  calcAtBottom: (dimensions) => {
+    const defaults = {
+      filterBarOffsetTop: 0,
+      filterBarOuterHeight: 0,
+      filterFormOuterHeight: 0,
+      mainNavHeight: 0,
+      windowTop: 0,
+    };
+    const values = Object.assign(defaults, dimensions);
+    const filterFormHeight = parseInt(values.filterFormOuterHeight, 10);
```

Comment:
I believe at some point we had something like `363px` getting passed in and this was meant to strip the units off. But looking at the jQuery results for both the `$.offset()` and `$.outerHeight()` functions, which should be populating this data, it's probably unnecessary. I'm fine with removing it, we just need to test it in multiple browsers (i.e. IE -- Ayeeeee!) to make sure everything still works as expected.